microk8s-resources/default-args/certs.d/localhost:32000/hosts.toml

```diff
@@ -0,0 +1,4 @@
+server = "http://localhost:32000"
+
+[host."http://localhost:32000"]
+  capabilities = ["pull", "resolve"]
```

Comment:
Please add missing new-line at the end of file

Reply:
done